### PR TITLE
Micrometer guide to send metric with Agent

### PR DESCRIPTION
**Pull request by estherk15**
### What does this PR do? What is the motivation?

- Give recommended options to send metrics through Datadog Agent when using Micrometer.
- [DOCS-6474](https://datadoghq.atlassian.net/browse/DOCS-6474)

### Merge instructions

[x]Please merge after reviewing

[DOCS-6474]: https://datadoghq.atlassian.net/browse/DOCS-6474?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by platinummonkey — APPROVED** [review]
👍 nice and simple, fantastic

File: content/en/metrics/guide/micrometer.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  tag: "Micrometer"
+  text: "Micrometer Prometheus"
+- link: "https://micrometer.io/docs/registry/prometheus"
+  tag: "Micrometer"
+  text: "Micrometer OTLP"
+---
+
+## Overview
+
+[Micrometer][1] is a vendor neutral interface that allows access to a metric with the ability to analyze across its dimensions. Often used with a Java Springboot application, it is used as an abstraction layer to submit metrics.
```
Comment (buraizu) [inline content/en/metrics/guide/micrometer.md:15]:
```suggestion
[Micrometer][1] is a vendor-neutral interface that allows access to metrics with the ability to analyze them across their dimensions. Often used with a Java Spring Boot application as an abstraction layer to submit metrics.
```

File: content/en/metrics/guide/micrometer.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+Micrometer offers multiple ways to integrate with Datadog. This guide outlines Datadog's recommended options to use with the Agent to send metrics to Datadog.
+
+## OpenTelemetry
+
+OTLP Ingestion by the Datadog Agent allows you to take advantage of observability feature in the Datadog Agent.
```
Comment (buraizu) [inline content/en/metrics/guide/micrometer.md:21]:
```suggestion
OpenTelemetry Protocol (OTLP) Ingestion by the Datadog Agent allows you to take advantage of observability feature in the Datadog Agent.
```

File: content/en/metrics/guide/micrometer.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
+---
+title: Send metrics with Micrometer
+kind: Guide
+further_reading:
+- link: "https://micrometer.io/docs/registry/otlp"
+  tag: "Micrometer"
+  text: "Micrometer Prometheus"
+- link: "https://micrometer.io/docs/registry/prometheus"
```
Comment (buraizu) [inline content/en/metrics/guide/micrometer.md:8]:
```suggestion
- link: "https://micrometer.io/docs/registry/prometheus"
  tag: "Micrometer"
  text: "Micrometer Prometheus"
- link: "https://micrometer.io/docs/registry/otlp"
```

Comment (buraizu) [reply content/en/metrics/guide/micrometer.md:8]: Not sure what the intended order is, but either the link or the text needs to be swapped for these two further reading resources

**Review by buraizu — APPROVED** [review]
Looks good overall, left some mostly minor updates, but there's a link mismatch in the further reading section

File: content/en/metrics/guide/micrometer.md (hunk 3 above)
Comment (estherk15) [reply content/en/metrics/guide/micrometer.md:8]: Thanks for catching this @buraizu!

**Pull request merged**